GeoEntity.md

```diff
@@ -0,0 +1,281 @@
+    
+	- Example URL:https://api.careerbuilder.com/core/geography/entities/admin_area_1s?country=us
+	
+- Response Sample:
+```
+{
+  "data": {
+    "entities": [
+      {
+        "long_name": "Alaska",
+        "short_name": "AK",
+        "level": 1,
+        "place_id": "ChIJG8CuwJzfAFQRNduKqSde27w",
+        "location": {
+          "lat": 64.2008413,
+          "lng": -149.4936733
+        },
+	"viewport": {
+          "northeast": {
+            "lat": 71.347066,
+            "lng": -129.9945563
+          },
+          "southwest": {
+            "lat": 51.220011,
+            "lng": 172.4573429
+          }
+        },
+        "formatted_address": "Alaska, USA"
+      },
+      ...
+    ]
+ }
+}
+```
+&nbsp;
+----------------------------
+#All Second-level Administrative Areas Request
+- Request Structure:
+	- Target: https://api.careerbuilder.com/core/geography/entities/admin_area_2s
+	- URL parameter: 
+  
+    | Parameter  | Required | Description |
+    |----------------|----------------|-----------------|
+    | country        | Yes | A country name or two-letter ISO-3166 country code, not casing sensitive            | 
+    | admin_area_one | Optional  | A first order administrative subdivision, both long name and short name are accepted. If present, limits the result set to the second-level administrative areas within the specified first-level administrative area. Otherwise, all second-level administrative areas within the specified country will be returned.|
+    
+	- Example URL: https://api.careerbuilder.com/core/geography/entities/admin_area_2s?country=us&admin_area_one=ga
+	
+- Response Sample:
+```
+{
+  "data": {
+    "entities": [
+      {
+        "long_name": "Camden County",
+        "short_name": "Camden County",
+        "level": 2,
+        "place_id": "ChIJl6NPkwkb5YgRwGvHFXP6tYY",
+        "location": {
+          "lat": 30.898276,
+          "lng": -81.6035062
+        },
+	"viewport": {
+          "northeast": {
+            "lat": 31.169595,
+            "lng": -81.40354649999999
+          },
+          "southwest": {
+            "lat": 30.7114626,
+            "lng": -81.936988
+          }
+        },
+        "formatted_address": "Camden County, GA, USA"
+      },
+      ...
+  ]
+ }
+}
+```
+&nbsp;
+----------------------------
+#All Localities Request
+- Request Structure:
+	- Target: https://api.careerbuilder.com/core/geography/entities/localities
+	- URL parameter: 
+  
+    | Parameter  | Required | Description |
+    |----------------|----------------|-----------------|
+    | country        | Yes | A country name or two-letter ISO-3166 country code, not casing sensitive            | 
+    | admin_area_one | Optional  | A first order administrative subdivision, both long name and short name are accepted. If present, limits the result set to the localities within the specified first-level administrative area. Otherwise, all localities within the specified country will be returned.|
```

Comment:
actually, I think in this context it flows well the way it's written @msundlof, I've written similar sentences in my own documentation before. the first sentence is structured similarly (technically it's a sentence fragment, but it works here). I don't know enough about grammar to say exactly *why* all this is, but I think in the context of technical documentation this style makes sense :)